scripts/serve-local.sh

```diff
@@ -9,6 +9,10 @@ docker run \
     --rm \
     --name janis \
     --tty --interactive \
-    --publish 3000:3000 \
-    --volume "$PWD:/app" \
+    --publish 3000:80 \
+    --volume "$PWD/src:/app/src" \
+    --volume "$PWD/public:/app/public" \
+    --volume "$PWD/package.json:/app/package.json" \
+    --env PORT=80 \
```

Comment:
While pairing with @ifsimicoded, we moved these to `Dockerfile` because we would always want these changes, not just when running this shell script. Just wanted to make note of it, doesn't seem necessary to make those changes in this PR.